admin_guide/managing_networking.adoc

```diff
@@ -1108,6 +1092,65 @@ to accept any connection allowed by each policy. That is,  connections on any
 port from pods in the *_same_* namespace, and connections on ports `80` and
 `443` from pods in *_any_* namespace.
 
+[[admin-guide-networking-networkpolicy-routers]]
```

Comment:
We probably should add a note about this to the router and to the routes sections with a link back to here.